beetmoverscript/templates/firefox_nightly_en_us_signed.yml

```diff
@@ -0,0 +1,20 @@
+{% include "firefox_nightly_en_us_base.yml" %}
+
+mapping:
+  # common deliverables
+{% for locale in ['en-US'] %}
+  {{ locale }}:
+  {% if platform == "linux-i686" %}
+    package:
+      artifact: target.tar.bz2
+      s3_key: firefox-{{ version }}.{{ locale }}.{{ platform }}.tar.bz2
+
+  {% endif %}
+
+  {% if platform == "linux-x86_64" %}
```

Comment:
are linux-x86_64 and linux-i686 broken into two blocks because we plan on adding more artifacts that might differ? Or should we say `if platform in ['linux-x86_64', 'linux-i686']` ?